test/e2e/pod_create_test.go

```diff
@@ -711,12 +711,14 @@ ENTRYPOINT ["sleep","99999"]
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
 
-		// container inside pod inherits user form infra container if --user is not set
-		// etc/passwd entry will look like 1000:*:1000:1000:container user:/:/bin/sh
+		u, err := user.Current()
+		Expect(err).ToNot(HaveOccurred())
+		// container inside pod inherits user from infra container if --user is not set
+		// etc/passwd entry will look like USERNAME:*:1000:1000:Full User Name:/:/bin/sh
 		exec1 := podmanTest.Podman([]string{"exec", ctrName, "cat", "/etc/passwd"})
 		exec1.WaitWithDefaultTimeout()
 		Expect(exec1).Should(Exit(0))
-		Expect(exec1.OutputToString()).To(ContainSubstring("container"))
+		Expect(exec1.OutputToString()).To(ContainSubstring(u.Name))
```

Comment:
Just out of principle, I think a better test would be:
```go
    .....{"exec", ctrName, "grep", "^"+u.Name+":", "/etc/passwd"})
```
or even (cleaner)
```go
    .....{"exec", ... , "id", u.Name})
```
Basically, to avoid a false match on username against (something else in passwd). Not that I can think of a real-world example, so this is paranoia. Anyone concur/disagree? Should I open an update/cleanup PR?

Reply:
we would probably catch it in the CI if the `passwd` file of any image breaks this test.  Said so, nothing against paranoia and fixing it earlier :-)